packages/lesswrong/server/resolvers/analyticsResolvers.ts

```diff
@@ -76,19 +76,24 @@ const queries: QueryFunc[] = [
   makePgAnalyticsQueryScalar({
     query: `
       SELECT
-        count(*) as unique_client_views_10_sec,
-        avg(max_reading_time) as median_reading_time
+        COUNT(*) AS unique_client_views_10_sec,
```

Comment:
Should this be checking for `total_reading_time > 10`? Or perhaps we should exclude bouncers from the `FROM` below so that they don't affect the `median_reading_time`? Not sure